<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OrdersTab</class>
 <widget class="QWidget" name="OrdersTab">
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QListWidget" name="list_orders">
     <property name="spacing">
      <number>12</number>
     </property>
     <property name="styleSheet">
      <string>QListWidget::item { margin: 5px; }</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btn_refresh">
       <property name="text">
        <string>Refresh Orders</string>
       </property>
       <property name="minimumHeight">
        <number>50</number>
       </property>
       <property name="font">
        <font>
         <pointSize>12</pointSize>
        </font>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_create_order">
       <property name="text">
        <string>Create New Order</string>
       </property>
       <property name="minimumHeight">
        <number>50</number>
       </property>
       <property name="font">
        <font>
         <pointSize>12</pointSize>
        </font>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>